mitxgraders/helpers/calc/calc.py

```diff
@@ -591,7 +620,7 @@ def handle_node(node):
             if any(np.any(np.isnan(r)) for r in as_list):
                 return float('nan')
 
-            return result
+            return cast_np_numeric_as_builtin(result)
```

Comment:
Shouldn't this be mapped across lists?

Reply:
I agree—good idea—and it's done. That said, I couldn't find a place where it mattered. (`result` should only be a list if it is the result of `eval_arguments`, and in that case all the list items should already have passed through `handle_node`). Anyway, I agree, but that's why there's no new test asserting this behavior.

Reply:
Agreed, I'm not sure it matters presently, but future-proofing!